build-system/tasks/visual-diff/helpers.js

```diff
@@ -65,61 +65,50 @@ function log(mode, ...messages) {
  *
  * @param {!puppeteer.Page} page a Puppeteer control browser tab/page.
  * @param {string} testName the full name of the test.
- * @param {!Array<string>} forbiddenCss Array of CSS elements that must not be
```

Comment:
In the old Ruby implementation, we used to differentiate between elements that must never be present on the page (`forbiddenCss`) and elements that are allowed to be present, but must eventually disappear (`loadingIncompleteCss`). 

I assume it isn't really necessary to do so in the current implementation? (The way you've reorganized things in this PR looks simpler, for sure.)

Reply:
We basically use the Forbidden case exactly once, which made me realize that it's a special case and doesn't require its own helper function. I moved that case into its own separate interactive test, located in the `fonts.amp.js` - the side effect is that it creates a new test (because every interactive test comes with a complement static test) but that's a minor side effect

Reply:
Fair enough.